### PR TITLE
Add @valentijnkap

### PR DESCRIPTION
**Pull request by valentijnkap**

### CODE REVIEW

**Comment by wooorm** [comment]
1. You’re missing an `index.js` and an `index.css`. Rename your `barchart.js` and `style.css` to use the previously mentioned filenames!
2. You should add yourself to the pinned list of GitHub handles in Slack
3. Seems you’re using git itself (which is awesome), but you should [link GitHub to Git](https://help.github.com/articles/setting-your-commit-email-address-in-git/)!

**Comment by wooorm** [comment]
Looks good, but could you remove `employee.csv` and `uren.csv`? Those aren’t part of this assignment.

Oh, and while your at it, could you lowercase `Dinners.csv` > `dinners.csv`?

**Comment by wooorm** [comment]
Note: I fixed the above comment to better reflect what’s lacking.

**Comment by valentijnkap** [comment]
I will remove employee.csv, that was dummy data. But uren.csv is the real data for my bar chart. I actually don't know why the csv, json, css and js files need to be the same as the index. There is some logic in those names.

**Comment by wooorm** [comment]
@valentijnkap Sorry, I thought you included only one assignment, but now I see that both the bar *and* the load assignment are handed in, so previous my [comment](https://github.com/cmda-fe3/course-17-18/pull/17#issuecomment-332575665) is misguided.

> I actually don't know why the csv, json, css and js files need to be the same as the index. There is some logic in those names.

First, because the load assignment asked for it! Second, for this course students are handing in roughly ± 750 assignments, and we’re checking some stuff automatically, so it makes sense to use similar names!

File: site/class-1-bar/valentijnkap/index.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+
+  <meta charset="utf-8">
+
+  <title>@valentijnkap | Uren gewerkt</title>
```
Comment (wooorm) [inline site/class-1-bar/valentijnkap/index.html]: Use *just* your `@ valentijnkap` here.

File: site/class-1-load/valentijnkap/index.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+<h1>Dinners</h1>
+
+<script src="http://d3js.org/d3.v3.min.js"></script>
+
+<script>
+
+	d3.json('index.json', function(data) {
+	 	console.log(data)
+	})
```
Comment (wooorm) [inline site/class-1-load/valentijnkap/index.html]: You’re loading the JSON, but not rendering it!

File: site/class-1-load/valentijnkap/index.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+	d3.csv('index.csv', function(data) {
+	 	console.log(data)
+	})
```
Comment (wooorm) [inline site/class-1-load/valentijnkap/index.html]: You don’t need this `d3.csv`, right?

File: site/class-1-load/valentijnkap/index.html [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+            .selectAll("tr")
+                .data(parsedCSV).enter()
+                .append("tr")
+
+            .selectAll("td")
+                .data(function(d) { return d; }).enter()
+                .append("td")
+                .text(function(d) { return d; });
+    })
```
Comment (wooorm) [inline site/class-1-load/valentijnkap/index.html]: 👍

File: site/class-1-load/valentijnkap/index.json [hunk 5]
```diff
@@ -0,0 +1,7 @@
+{
+	"Dinners": {
+		"24-09-2017": "Rusty",
+		"23-09-2017": "McChicken",
+		"22-09-2017": "Moink Balls"
+	}
+}
```
Comment (wooorm) [inline site/class-1-load/valentijnkap/index.json]: Currently, your JSON is an object.

It makes more sense to use an array for your data, such as this:

```js
[
  {
    "day": "24-09-2017",
    "dinner": "Rusty"
  },
  // ...
]
```

...and that also makes it much easier to render!

**Comment by wooorm** [comment]
Awesome! Well done!

**Comment by wooorm** [comment]
P.S. I saw you used `http` for d3 instead of `https`, and fixed it for you (and there’s now a check for it too)!

**Pull request merged**